packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageModule.kt

```diff
@@ -42,4 +52,62 @@ class ExpoImageModule(val context: ReactApplicationContext) : ReactContextBaseJa
       }
     }
   }
+
+  @ReactMethod
+  fun queryCache(urls: ReadableArray, promise: Promise) {
+    moduleCoroutineScope.launch(
+        CoroutineExceptionHandler { _, e ->
+          promise.reject("ERR_IMAGE_QUERY_CACHE_FAILURE", "Failed to read the cache: ${e.message}", e)
+        }
+    ) {
+      val resultMap = WritableNativeMap()
+      urls.toArrayList()
+        .filterIsInstance<String>()
+        .map { url -> async {
+          val isInDiskCache = isInCache(url, ImageCacheType.DISK)
+          val isInMemoryCache = isInCache(url, ImageCacheType.MEMORY)
+          Triple(url, isInDiskCache, isInMemoryCache)
+        }
+        }
```

Comment:
These fragment made me anxious (L70-L71):
```kotlin
}
}
```
```suggestion
        .map { url ->
          async {
            val isInDiskCache = isInCache(url, ImageCacheType.DISK)
            val isInMemoryCache = isInCache(url, ImageCacheType.MEMORY)
            Triple(url, isInDiskCache, isInMemoryCache)
          }
        }
```

Reply:
I recommend running `./gradlew :expo-image:spotlessApply` in `bare-expo/android` directory - it auto-formats code. 

P.S. New IntelliJ IDEA has now built-in plugin `Actions on save` - hopefully Android Studio will have it soon. It can be configured to autoformat file with Kotlin Style Guide rules on each Cmd+S 😁